pkg/auth/dependency/mfa/authenticator.go

```diff
@@ -171,3 +171,42 @@ func CanAddAuthenticator(authenticators []interface{}, newA interface{}, mfaConf
 
 	return true
 }
+
+func IsDeletingLastActivatedAuthenticator(authenticators []interface{}, a interface{}) bool {
```

Comment:
I think naming it `IsDeletingOnlyActivatedAuthenticator` would be better. My first impression is that it means the authenticator activated the latest ('last activated')